snapshots/MinimalDelegationExecuteTest.json

```diff
@@ -1,10 +1,10 @@
 {
-  "execute_BATCHED_CALL_SUPPORTS_OPDATA_singleCall": "92380",
-  "execute_BATCHED_CALL_SUPPORTS_OPDATA_twoCalls": "129072",
-  "execute_BATCHED_CALL_opData_P256_singleCall": "110657",
-  "execute_BATCHED_CALL_opData_singleCall": "92380",
-  "execute_BATCHED_CALL_opData_singleCall_native": "93430",
-  "execute_BATCHED_CALL_opData_twoCalls": "129072",
+  "execute_BATCHED_CALL_SUPPORTS_OPDATA_singleCall": "92373",
```

Comment:
oops i think main gas might be off, I'm on latest nightly and ran using --isolate